meson_options.txt

```diff
@@ -127,3 +127,8 @@ option('protobuf-3-6-0',
        type: 'boolean',
        value: false,
        description: 'Use the protobuf 3.6.0 subproject')
+
+option('embed',
+       type: 'boolean',
```

Comment:
What about making this a `string` option with the file to embed?

Reply:
I thought of that first, but then I realized that you compile against one exact header and const name from the real source (`loader.cc`). Then the build process must supply such header and symbol. The original data file name could be changed though, having meson pass the header name to the embedder.py as a new second parameter. That way the builder calls with `-Dembed_weights=/my/weights` and meson calls embedder.py with the real filename and the fixed header name for the source (for weights). Later maybe another `-Dembed_config=/my/config`